script.js

```diff
@@ -0,0 +1,144 @@
+// funtion reacts to button
+
+const myFunc = () => {
+
+    cityApi = document.getElementById('europe').value
```

Comment:
Missing a `const` here maybe?
```suggestion
    const cityApi = document.getElementById('europe').value
```